yaksh/templates/yaksh/monitor.html

```diff
@@ -98,11 +111,16 @@
         <td> {{ paper.user.username }} </td>
         <td> {{ paper.user.profile.roll_number }} </td>
         <td> {{ paper.user.profile.institute }} </td>
-        <td> {{ paper.get_answered_str }} </td>
+        <td> {% for que in paper.get_attempted_questions %}
+             <a data-toggle="tooltip" title="{{que.description|striptags}}" data-item-id="{{paper.user.id}}_{{que.id}}_{{paper.id}}" class="item">
+             {{ que.id }}
```

Comment:
Is the moderator expected to remember the question id? Or should he click on the ids everytime he has to search for a question. Maybe instead of question id, it can show the question description.